packages/devtools_app/lib/src/inspector/diagnostics.dart

```diff
@@ -82,9 +82,18 @@ class DiagnosticsNodeDescription extends StatelessWidget {
         return;
       }
     }
+
     if (description?.isNotEmpty == true) {
       yield TextSpan(text: description, style: textStyle);
     }
+
+    final textPreview = diagnostic.json['textPreview'];
+    if (textPreview is String) {
```

Comment:
should we be checking for `is String` or `!= null` ?

Reply:
I now need to check it's a string to strip newlines from it